run_container.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/coreos/go-systemd/dbus"
+)
+
+// Public Functions
+
+// RunContainer runs a container with given ID
+func RunContainer(containerID string, containerPath string) error {
+	conn, err := dbus.NewSystemConnection()
+	if err != nil {
+		return err
+	}
+
+	return runContainerInternal(containerID, containerPath, conn)
+}
+
+// Internal Functions
+
+// Produce a per-container unique name used to identify its transient systemd service
+func getUnitName(containerID string) string {
+	return fmt.Sprintf("kpod-container-%s.service", containerID)
```

Comment:
kpod should create pods not containers. At least this is what I thought we agreed on.